src/components/SharedComponents/Playfield.vue

```diff
@@ -6,12 +6,13 @@
         <h3 style="text-align: left; margin-left: 40px" :style="playfieldTextColour()">playerStacks() {</h3>
       </div>
     </div>
-    <div class="row">
-        <div class="stacks col-lg-3 col-md-4 col-sm-6" v-for="stack in stacks">
+    <div class="row my-4">
```

Comment:
If you changed the class, shouldn't there also be a style file update in this commit?

Reply:
Resolved